include/fcl/geometry/bvh/BVH_model.h

```diff
@@ -40,6 +40,7 @@
 
 #include <vector>
 #include <memory>
+#include <new>
```

Comment:
Nit: It would be good to move this to `BVH_model-inl.h` as `BVH_model-inl.h` actually uses this header.